python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -947,11 +947,11 @@ def __defaultActiveBackendsFunc(self):
                 j = stripProxy(self.registry_slice(i))
 
                 job_status = lazyLoadJobStatus(j)
-                backend_obj = lazyLoadJobBackend(j)
-                backend_name = getName(backend_obj)
 
                 if job_status in ['submitted', 'running'] or (j.master and (job_status in ['submitting'])):
                     if self.enabled is True and self.alive is True:
+                        backend_obj = lazyLoadJobBackend(j)
```

Comment:
@rob-c It would be better to not create the object as we only need a name but I didn't want to interfere too much as I didn't want to risk breaking lazy loading. This could be done in another PR I guess.